src/rebuild/srv.c

```diff
@@ -1259,12 +1259,17 @@ rebuild_task_complete_schedule(struct rebuild_task *task, struct ds_pool *pool,
 
 
 	/* Update the rebuild complete status for pool query */
-	rc1 = rebuild_status_completed_update(task->dst_pool_uuid, &rgt->rgt_status);
-	if (rc1 != 0) {
-		D_ERROR("rebuild_status_completed_update, "DF_UUID" failed: "DF_RC"\n",
-			DP_UUID(task->dst_pool_uuid), DP_RC(rc1));
-		if (rc == 0)
-			rc = rc1;
+	/* Do not need update rebuild status for reclaim job, which is to cleanup the current job and
```

Comment:
(style)  line over 100 characters

Reply:
simple to fix this one.  If you install clang-format 14.05+, clang can automatically reformat your code in the githook

Reply:
ah, sure, thanks